server/views/pages/adjudicationForReport/punishmentsTab.njk

```diff
@@ -24,7 +24,9 @@
   {{ adjudicationTabHeaderSingular(reportNo, reviewStatus | reportedAdjudicationStatusDisplayName, prisoner, 'punishments', reportHref, hearingsHref, punishmentsHref) }}
 
   {% if chargeProved %}
-    {{ hearingDamagesAndCautionSummary(reportNo, moneyRecoveredBoolean, moneyRecoveredAmount, caution, { "data-qa": "money-caution-summary" }, readOnly, moneyChangeLinkHref, cautionChangeLinkHref) }}
+    {% if showEditableDamagesAndCautionDetails %}
+      {{ hearingDamagesAndCautionSummary(reportNo, moneyRecoveredBoolean, moneyRecoveredAmount, caution, { "data-qa": "money-caution-summary" }, readOnly, moneyChangeLinkHref, cautionChangeLinkHref) }}
```

Comment:
There is another instance of `hearingDamagesAndCautionSummary` on this page which is displayed if `ReportedAdjudicationStatus.QUASHED` and this has `readOnly = true`, haven't updated that